docs/Webhooks/Geo-triggering.md

```diff
@@ -1,183 +1,207 @@
-Set the type to trigger in the request.
+### `deviceInfo` fields
 
-Endpoint: https://config.bluedot.io/prod1/projects
+This table lists the fields within the `deviceInfo` object, containing details about the device.
 
-Example of adding a Geo-trigger Webhook:
-```json
-"webhooks": [{
-    "type": "trigger",
-    "url": "<webhook URL goes here>"
-}]
-```
+| Field             | Description                           |
+|-------------------|---------------------------------------|
+| `deviceInfo.deviceType` | Type of device (e.g., iPhone14,7). |
+| `deviceInfo.os`   | Operating system of the device.       |
+| `deviceInfo.osVersion` | Operating system version of the device. |
 
-Webhook request JSON structure
-------------------------------
+### `zoneInfo` fields
 
-### Entry JSON request
-```json
-{
-    "type": "checkIn",
-    "checkInId":"73cbdf0c-5523-46ec-bc15-c9ad362b2a85",
-    "installRef":"c92f4a32dc35282d4471b42993f809fa",
-    "checkInTime": "02-09-2018 00:08:16",
-    "checkInTimeISO": "2018-09-02T00:08:16.000Z",
-    "longitude":144.98173087835312,
-    "latitude":-37.819805462370944,
-    "fenceName":"Melbourne Cricket Ground Gate 1",
-    "deviceSpeed":10,
-    "fenceId":"80fc36ad-ee72-4450-ad96-b3fadfc26cb4",
-    "zoneName":"Melbourne Cricket Ground",
-    "zoneId":"ffece0a9-fd21-4148-892e-0a61d01a6bd4",
-    "deviceType":"iPhone 9,3",
-    "sdkVersion": "1.13.0",
-    "os": "iOS",
-    "osVersion": "12.1.2",
-    "appBuildVersion": "2.8.0.639",
-    "receivedAt": "2018-09-02T00:22:43.816Z"
-}
-```
+This table provides information about the `zoneInfo` object, including details about the zone and destination.
 
-### Entry JSON request with `customEventMetaData` and `customData`
-```json
-{
-  "type": "checkIn",
-  "checkInId":"73cbdf0c-5523-46ec-bc15-c9ad362b2a85",
-  "installRef":"c92f4a32dc35282d4471b42993f809fa",
-  "checkInTime": "02-09-2018 00:08:16",
-  "checkInTimeISO": "2018-09-02T00:08:16.000Z",
-  "longitude":144.98173087835312,
-  "latitude":-37.819805462370944,
-  "fenceName":"Melbourne Cricket Ground Gate 1",
-  "deviceSpeed":10,
-  "fenceId":"80fc36ad-ee72-4450-ad96-b3fadfc26cb4",
-  "zoneName":"Melbourne Cricket Ground",
-  "zoneId":"ffece0a9-fd21-4148-892e-0a61d01a6bd4",
-  "deviceType":"iPhone 9,3",
-  "sdkVersion": "1.13.0",
-  "appBuildVersion": "2.8.0.639",
-  "os": "iOS",
-  "osVersion": "12.1.2",
-  "receivedAt": "2018-09-02T00:22:43.816Z",
-  "zoneCustomData": {
-      "key1": "value1",
-      "key2": "value2"
-  },
-  "eventMetaData": {
-      "eKey1": "eValue1"
-  }
-}
-```
+| Field                                 | Description                                                 |
+|---------------------------------------|-------------------------------------------------------------|
+| `zoneInfo.customData`                 | Custom data related to the zone.                            |
+| `zoneInfo.customData.[KEY]`         | The key of the zone custom data key/value pair.                    |
```

Comment:
Similar to above, should be value, rather than key.

Reply:
Updated!